package.json

```diff
@@ -9,14 +9,14 @@
     "Andrew Monks <a@monks.co>"
   ],
   "dependencies": {
-    "@blueoak/list": "^9.0.0",
-    "@npmcli/arborist": "^6.5.0",
+    "@blueoak/list": "^15.0.0",
+    "@npmcli/arborist": "^7.5.4",
     "correct-license-metadata": "^1.4.0",
     "docopt": "^0.6.2",
     "hasown": "^2.0.0",
     "npm-license-corrections": "^1.6.2",
     "semver": "^7.6.0",
-    "spdx-expression-parse": "^3.0.1",
+    "spdx-expression-parse": "^4.0.0",
```

Comment:
just noting that https://github.com/jslicense/spdx-expression-parse.js/blob/main/package.json doesnt declare its minimum supported node version